webapp/channels/src/components/root/root.tsx

```diff
@@ -61,68 +48,38 @@ import RootRedirect from './root_redirect';
-const LazySelectTeam = React.lazy(() => import('components/select_team'));
-const LazyAuthorize = React.lazy(() => import('components/authorize'));
-const LazyCreateTeam = React.lazy(() => import('components/create_team'));
-const LazyMfa = React.lazy(() => import('components/mfa/mfa_controller'));
-const LazyPreparingWorkspace = React.lazy(() => import('components/preparing_workspace'));
-const LazyTeamController = React.lazy(() => import('components/team_controller'));
-const LazyOnBoardingTaskList = React.lazy(() => import('components/onboarding_tasklist'));
-
-const CreateTeam = makeAsyncComponent('CreateTeam', LazyCreateTeam);
-const ErrorPage = makeAsyncComponent('ErrorPage', LazyErrorPage);
-const TermsOfService = makeAsyncComponent('TermsOfService', LazyTermsOfService);
-const Login = makeAsyncComponent('LoginController', LazyLogin);
-const AdminConsole = makeAsyncComponent('AdminConsole', LazyAdminConsole);
-const LoggedIn = makeAsyncComponent('LoggedIn', LazyLoggedIn);
-const PasswordResetSendLink = makeAsyncComponent('PasswordResedSendLink', LazyPasswordResetSendLink);
-const PasswordResetForm = makeAsyncComponent('PasswordResetForm', LazyPasswordResetForm);
-const Signup = makeAsyncComponent('SignupController', LazySignup);
-const ShouldVerifyEmail = makeAsyncComponent('ShouldVerifyEmail', LazyShouldVerifyEmail);
-const DoVerifyEmail = makeAsyncComponent('DoVerifyEmail', LazyDoVerifyEmail);
-const ClaimController = makeAsyncComponent('ClaimController', LazyClaimController);
-const LinkingLandingPage = makeAsyncComponent('LinkingLandingPage', LazyLinkingLandingPage);
-const SelectTeam = makeAsyncComponent('SelectTeam', LazySelectTeam);
-const Authorize = makeAsyncComponent('Authorize', LazyAuthorize);
-const Mfa = makeAsyncComponent('Mfa', LazyMfa);
-const PreparingWorkspace = makeAsyncComponent('PreparingWorkspace', LazyPreparingWorkspace);
-const TeamController = makeAsyncComponent('TeamController', LazyTeamController);
-const OnBoardingTaskList = makeAsyncComponent('OnboardingTaskList', LazyOnBoardingTaskList);
-
-type LoggedInRouteProps = {
-    component: React.ComponentType<RouteComponentProps<any>>;
-    path: string | string[];
-    theme?: Theme; // the routes that send the theme are the ones that will actually need to show the onboarding tasklist
-};
-
-function LoggedInRoute(props: LoggedInRouteProps) {
-    const {component: Component, theme, ...rest} = props;
-    return (
-        <Route
-            {...rest}
-            render={(routeProps) => (
-                <LoggedIn {...routeProps}>
-                    {theme && <CompassThemeProvider theme={theme}>
-                        <OnBoardingTaskList/>
-                    </CompassThemeProvider>}
-                    <Component {...(routeProps)}/>
-                </LoggedIn>
-            )}
-        />
-    );
-}
+const MobileViewWatcher = makeAsyncComponent('MobileViewWatcher', lazy(() => import('components/mobile_view_watcher')));
```

Comment:
We might want to take a second pass over which of these that we're lazy loading. Some of these important layout components like the `GlobalHeader`, `TeamSidebar`, `ModalController`, and `LoggedIn` should probably not be split out because they're core part of the app which you'd want visible immediately

Reply:
Reverted for GlobalHeader and LoggedIn. But for TeamSidebar since its conditional based on teams i have kept it lazy and same for ModalController which isnt required at the start of the app.

Reply:
I think you might've missed the commit that reverted `GlobalHeader` since that's still lazy.

That said, I didn't realize that `LoggedIn` and `TeamController` were already loaded async before. That still feels wrong to me to do that, but that wouldn't be a new change here

Reply:
oh yeah i reverted GlobalHeader and Loggedin. TeamController i can we have still have it lazy since user could land on other routes as well

Reply:
They can land on other routes, but I think we should optimize for landing in a team/channel since that's where I'd expect them to be most of the time